This patch is needed for building the package as of F-11.  It can be
dropped whenever autoconf 2.63 is no longer used on any live branch.


diff -Naur tiff-3.9.2.orig/configure.ac tiff-3.9.2/configure.ac
--- tiff-3.9.2.orig/configure.ac	2009-11-04 12:11:20.000000000 -0500
+++ tiff-3.9.2/configure.ac	2009-12-03 12:52:41.000000000 -0500
@@ -24,7 +24,7 @@
 
 dnl Process this file with autoconf to produce a configure script.
 
-AC_PREREQ(2.64)
+AC_PREREQ(2.63)
 AC_INIT([LibTIFF Software],[3.9.2],[tiff@lists.maptools.org],[tiff])
 AC_CONFIG_AUX_DIR(config)
 AC_CONFIG_MACRO_DIR(m4)
